scripts/script.js

```diff
@@ -1,7 +1,7 @@
 // main js file
 'use strict';
 
-var allProjects = [];
+// var allProjects = []; OLD ARRAY
```

Comment:
Just remove it if it's old. You can always go back and reference it in your git history.